### DISCUSSION TITLE
LVGL on an esp32s3-qio_opi

### DISCUSSION THREAD

**Discussion by jens-schiffke** [arendst/Tasmota#20705] [Support request]
Can you please provide a firmware for an ESP32-4848S040 with LVGL at "https://ota.tasmota.com/tasmota32/release/"?
Greetings, Jens

**Comment by sfromis** [comment]
For LVGL with those chips, I'm using `tasmota32s3cdc-qio_opi` or `tasmota32s3-qio_opi` dev builds from https://github.com/tasmota/install/tree/firmware/firmware/unofficial

**Reply by jens-schiffke** [reply]
Thank you!
After uploading a display.ini (https://github.com/arendst/Tasmota/discussions/20527#discussioncomment-8197340) "ST7701" appears on the display and the device is no longer responsive.

**Reply by sfromis** [reply]
Having just the `display.ini` does not give you anything interesting to see. And beware that your power supply may not be up to the job when having the display.
